src/components/Picklist/index.js

```diff
@@ -0,0 +1,401 @@
+import React, { Component } from 'react';
```

Comment:
Instead of maintain activeChildren and currentOption what you can do is register and unregister when in child when is deselected and selected in componentDidUpdate in PicklistOption component. Try this since it will simply all logic related to activeChildren